src/lints/struct_now_doc_hidden.ron

```diff
@@ -0,0 +1,50 @@
+SemverQuery(
+    id: "struct_now_doc_hidden",
+    human_readable_name: "pub struct is now #[doc(hidden)]",
+    description: "A pub struct is now hidden from documentation effectively removing it from the crate's public api.",
```

Comment:
Minor wording tweak suggestion:
```suggestion
    description: "A pub struct is now marked #[doc(hidden)] and is thus no longer part of the public API.",
```

I wanted to avoid the "effectively" since it's a bit of a weasel word and makes it sound like a "lesser" semver issue. `doc(hidden)` not being public API by convention is less known but is an hard fact of how Rust works, as the reference link you added shows.

Reply:
Makes perfect sense, thanks!